ntbs-service/Migrations/20200206121302_AddLegacyCountries.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations
+{
+    public partial class AddLegacyCountries : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<bool>(
+                name: "LegacyCountry",
```

Comment:
`IsLegacy`? `LegacyCountry` sounds a bit like a column where you'd find a reference to some other entity